src/platform/user/profile/vet360/actions/transactions.js

```diff
@@ -7,7 +7,6 @@ import { showAddressValidationModal } from '../../utilities';
 import localVet360, { isVet360Configured } from '../util/local-vet360';
 import { CONFIRMED } from '../../constants/addressValidationMessages';
 import {
-  addCountryCodeIso3ToAddress,
```

Comment:
this helper is no longer needed as we are setting the ISO3 code directly via the edit form.